WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/GutenbergLayoutFilterBar.swift

```diff
@@ -0,0 +1,67 @@
+import UIKit
+
+protocol FilterBarDelegate {
+    func numberOfFilters() -> Int
+    func filter(forIndex: Int) -> GutenbergLayoutSection
+    func willSelectFilter(forIndex: Int)
```

Comment:
The delegate doesn't actually use the index yet but from an API design perspective I thought it was better to include it.